### PR TITLE
Update IRC information in readme and config

### PR DESCRIPTION
**Pull request by aurorasmiles**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ High performance Spigot fork that aims to fix gameplay and mechanics inconsisten
 
 
 **Support and Project Discussion:**
- - [IRC](https://irc.spi.gt/iris/?channels=paper) or [Discord](https://discord.gg/papermc)
+ - IRC #paper on irc.esper.net or [Discord](https://discord.gg/papermc)
```
Comment (Proximyst) [inline README.md]:
```suggestion
 - IRC `#paper` on `irc.esper.net` or [Discord](https://discord.gg/papermc)
```

**Review by Proximyst — DISMISSED** [review]
Looks pretty good. Small nitpick.

**Review by Proximyst — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ High performance Spigot fork that aims to fix gameplay and mechanics inconsisten
 
 
 **Support and Project Discussion:**
- - [IRC](https://irc.spi.gt/iris/?channels=paper) or [Discord](https://discord.gg/papermc)
+ - IRC `#paper` on `irc.esper.net` or [Discord](https://discord.gg/papermc)
```
Comment (kashike) [inline README.md]: https://webchat.esper.net/?channels=paper

File: Spigot-Server-Patches/0002-Paper-config-files.patch [hunk 3]
```diff
@@ -307,7 +307,7 @@ index 0000000000000000000000000000000000000000..2d9b6439cff9e8907b55c8882b15e94a
 +            + "join us in our Discord or IRC channel.\n"
 +            + "\n"
 +            + "Discord: https://discord.gg/papermc\n"
-+            + "IRC: #paper @ irc.spi.gt ( http://irc.spi.gt/iris/?channels=paper )\n"
++            + "IRC: #paper @ irc.esper.net \n"
```
Comment (kashike) [inline Spigot-Server-Patches/0002-Paper-config-files.patch]:
```suggestion
+            + "IRC: #paper @ irc.esper.net ( https://webchat.esper.net/?channels=paper ) \n"
```

**Review by kashike — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (aurorasmiles) [reply README.md]: thx

**Review by kashike — APPROVED** [review]

**Pull request merged**